src/style.scss

```diff
@@ -162,6 +162,10 @@ textarea {
   color: var(--text-color) !important;
 }
 
+.text-black {
```

Comment:
Is this being used? Ideally we don't want to hardcode colors like because each space may have different colors, for example on Balancer space a black text may not be visible cuz the theme have black background.

Reply:
Right! I removed all hard coded styles